loopdb/store_mock.go

```diff
@@ -165,6 +165,12 @@ func (s *StoreMock) FetchLoopInSwaps(ctx context.Context) ([]*LoopIn,
 	return result, nil
 }
 
+func (s *StoreMock) UpdateLoopOutAssetInfo(ctx context.Context,
+	hash lntypes.Hash, asset *LoopOutAssetSwap) error {
+
+	return nil
```

Comment:
Is the mock implementation expected to handle assets? 

I propose to return an error here ("unimplemented").